src/dances/bar.js

```diff
@@ -0,0 +1,23 @@
+export default (elem, value, options = {}) => {
+  const from = options.from || [0,0,0]
+  const to = options.to || [255,255,255]
+  const scaleR = (to[0] - from[0]) * value
+  const scaleG = (to[1] - from[1]) * value
+  const scaleB = (to[2] - from[2]) * value
+  var color = `rgb(${Math.floor(to[0] - scaleR)}, ${Math.floor(to[1] - scaleG)}, ${Math.floor(to[2] - scaleB)})`
+  elem.style.position='relative'
+
+  var rip1 = null
+  if (elem.childNodes.length == 0) {
+    rip1 = document.createElement("div");
```

Comment:
In my opinion rythm should only act on a few css property and should not take care of creating elements

Reply:
I understand. But I am not getting any different ideas on bar. It seems to me there has to be a container and the actual bar which will be aligned at bottom of the container. 

So should we expect user to give both elements by him/herself ? like -
```html
<div class="rythm bar">
    <div class="bar-child"></div>
</div>
```
May be not decent class names but you get the idea.

Reply:
I think it would be more flexible if it only change the height, it will fit more use case. If someone want to archieve your example, he will have to make the container. This dance will only change height of the element.
I think what we can do is renaming this effect to "height" to match the property name, and make an example of how to do your example ? (Making a container + using height dance + using color dance)
What do you think ?

Reply:
I think thats better 👍 . I will change accordingly and submit back once I get time.